app/controllers/annotation_controller.js

```diff
@@ -0,0 +1,168 @@
+import Annotation from '../models/annotation';
+
+export const createAnnotation = (req, res) => {
+  if (req.isAuthenticated()) {
```

Comment:
That would be cleaner, but in some cases, I think we were planning on allowing an unauthenticated request to still go through. For instance, a user who isn't logged in may still be able to get public annotations.